### DISCUSSION TITLE
opensumi如何使用toolbar？

### DISCUSSION THREAD

**Discussion by hanbingzi** [opensumi/core#2688] [Q&A]
我想在opensumi中做一个toolbar，类似于下面图片的这种，我查看了opensumi core项目，发现有个toolbar应该是用来实现此功能的，但我找不到文档去学习如何使用toolbar，希望有心人帮忙解答下。
<img width="1728" alt="image" src="https://user-images.githubusercontent.com/20720644/237036470-7cf2fa33-9921-4b8b-903f-ace588d1e371.png">

**Comment by bytemain** [comment]
参考以下两个讨论：
- https://github.com/opensumi/core/discussions/2668
- https://github.com/opensumi/core/discussions/2573

**Comment by erha19** [comment]
模块拓展案例:

https://github.com/opensumi/opensumi-module-samples/blob/main/modules/connection/README.md

插件拓展案例：

https://github.com/opensumi/opensumi-extension-samples/tree/main/toolbar-sample

@hanbingzi 可以参考一下

**Reply by hanbingzi** [reply]
谢谢，我用的模块拓展案例，但是我在SlotLocation.top中插入的react组件，使用opensumi自带的select，下拉无法显示，应该是被遮挡了，请问通常这样情况应该如何解决？
![image](https://user-images.githubusercontent.com/20720644/237058144-048417e1-770a-450f-8caf-7c837972f882.png)

**Comment by erha19** [comment]
@hanbingzi 2.24.2 版本会修复这个层级问题，暂时，你也可以手动修改 Toolbar样式，将其层级提升，或用我们目前的测试版本 `2.24.2-next-1683369127.0`.

相关 PR：https://github.com/opensumi/core/pull/2670

**Reply by hanbingzi** [reply]
好的，谢谢
